Cpp_test/lib/ipc-pipe-send.cpp

```diff
@@ -18,8 +18,20 @@ void IPCPipeSend::init()
 		throw std::runtime_error(std::string("ERROR: create pipe with send name ") + this->info.method_name);
 	}
 
-	// open the pipe
-	this->pd = open(this->info.method_name, O_WRONLY);
+	int time_wait{0};
```

Comment:
Missing the waiting print.
```suggestion
        std::cout << "Waiting for receiver... " << std::endl;
	int time_wait{0};
```